pages/index.vue

```diff
@@ -87,6 +96,7 @@ const getServiceDescription = (serviceName: string) => {
   const descriptions = {
     Superset: t('services.supersetDescription'),
     Windmill: t('services.windmillDescription'),
+    "Dataset Explorer": t('services.explorerDescription'),
```

Comment:
```suggestion
    "Explorer": t('services.explorerDescription'),
```

And please adjust i18n